docs/mosip-installation-without-public-dns.md

```diff
@@ -1,38 +1,45 @@
 # MOSIP installation without public DNS
 
-## Prerequisites
-* [openssl](https://www.openssl.org/)
-
-## Nginx setup
-* Generate self-signed certificate for domain `*.sandbox.xyz.net`.
-    * Update the below variables based on requirements.
+## Nginx node setup
+### Follow the below steps before installing nginx
```

Comment:
Shouldnt this heading be shorter?  Steps before installing Nginx  (you can think of anything else as well)